packages/clerk-js/src/core/clerk.ts

```diff
@@ -1045,6 +1046,13 @@ export default class Clerk implements ClerkInterface {
 
   #clearJustSynced = () => removeClerkQueryParam(CLERK_SYNCED);
 
+  #isReturningFromPrimary = () => getClerkQueryParam(CLERK_REFERRER_PRIMARY) === 'true';
+
+  #clearPrimaryReferrer = () => {
```

Comment:
@panteliselef 🙃 the function name should indicate that it sets another query param.